tools/graph_generation/grapher.r

```diff
@@ -0,0 +1,63 @@
+install.packages("ggplot2")
+library("ggplot2")
+library("plyr")
+
+setwd("C:\\Users\\tirpi\\Downloads")
+ue4_stats <- read.csv("ue4_stats.csv", row.names = NULL)
+acl_stats <- read.csv("acl_stats.csv", row.names = NULL)
```

Comment:
or perhaps read those two paths from the command line (one, or the other, or both perhaps?)